src/DataAccess/PaymentMigration/InsertingPaymentHandler.php

```diff
@@ -43,9 +43,10 @@ private function flush(): void {
 		$this->entityManager->clear();
 
 		$this->startTransactionIfNeeded();
-		$stmt = $this->entityManager->getConnection()->prepare( "UPDATE request SET payment_id=? WHERE id=?" );
+		$statement = $this->entityManager->getConnection()->prepare( "UPDATE request SET payment_id=? WHERE id=?" );
 		foreach ( $this->paymentIdCollection as $membershipId => $paymentId ) {
-			$stmt->executeQuery();
+			$statement->bindValue( $paymentId, $membershipId );
```

Comment:
Looking at the [docs](https://www.doctrine-project.org/projects/doctrine-dbal/en/4.0/reference/data-retrieval-and-manipulation.html#dynamic-parameters-and-prepared-statements) for this, I think you need to bind the values separately:

```php
$statement->bindValue( 1, $paymentId );
$statement->bindValue( 2, $membershipId );
```